support/ccm.rb

```diff
@@ -1010,9 +996,9 @@ def create_cluster(name, version, datacenters, nodes_per_datacenter)
     config << 'max_hints_delivery_threads: 1'
 
     # If we're just dealing with C* 4.0 enable MV as well
-    if cassandra_version >= '4.0'
-      config << 'enable_materialized_views: true'
-    end
+    #if cassandra_version >= '4.0'
+    #  config << 'enable_materialized_views: true'
+    #end
```

Comment:
This was causing problems on newer versions of DSE but may still apply for C* 4.0 and (barely) lower.  This might need more investigation before we call this done.

Reply:
Seems fine with both C* 4.0.0-alpha3 and 3.11.6.  Calling this good for now, can revisit if we see problems going forward

Reply:
Addressed in https://github.com/datastax/ruby-driver/commit/d7cb69c47a6d0c11385a6792f45c976b32066200